### ISSUE TITLE
Where is MobileNeRF -> Asset Downloads 

### ISSUE THREAD

**Issue by yosun** [julienkay/MobileNeRF-Unity-Viewer#11]
MobileNeRF -> Asset Downloads does not seem to exist?

Import From Disk seems to be the only option?

**Comment by Dariushuangg** [comment]
My guess is that you didn't drag the Editor folder in the package into "Asset", so the editor scripts are not compiled by Unity Editor.

**Comment by yosun** [comment]
clean install, empty project.
which version of unity should i be using - which has this been tested to work? 

[![Image from Gyazo](https://i.gyazo.com/c2de7ca3510f74bf382ba316b6352fb1.png)](https://gyazo.com/c2de7ca3510f74bf382ba316b6352fb1)

**Comment by yosun** [comment]
no errors on console - screenshot shown

i'm also not sure what types of files import from disk works with?

**Comment by Dariushuangg** [comment]
Based upon your screenshot, I am pretty sure what's wrong. Please read my previous comment again and I think your issue will be resolved.

**Comment by Dariushuangg** [comment]
Try doing what I said and take another screenshot?

**Comment by yosun** [comment]
hmm i did that too but no effect earlier... after re-import, same thing? which version has this been tested with? 

[![Image from Gyazo](https://i.gyazo.com/cda2e38b63b51db68efb588bc23a8585.png)](https://gyazo.com/cda2e38b63b51db68efb588bc23a8585)

**Comment by Dariushuangg** [comment]
Downloading works for me on 2021.3.16 with URP.

**Comment by yosun** [comment]
Is URP required? I use builtin by default...

**Comment by Dariushuangg** [comment]
I don't think so, since the error seems to be in the Editor and not in rendering...Maybe just start a new project in the version I provided, install package, and move the Editor folder to the Asset directory.

**Comment by julienkay** [comment]
@yosun 
Thanks for providing the screenshots. The Unity version you're using should work.
I see that you're using macOS though, which I've never tested this with before. If anyone else used it with macOS successfully or can reproduce this issue feel free to chime in.

Otherwise I have to assume that this is a Unity bug, where the menu items are not displayed correctly on macOS.

If I had to take a stab in the dark: In MobileNeRFImporter.cs can you try and modify all occurrences of 
[MenuItem("MobileNeRF/Asset Downloads/...", ..., ...)] and delete the last two parameters so it's only
[MenuItem("MobileNeRF/Asset Downloads/..."]

Maybe also try to delete some of the fluff like lines 28-59?
https://github.com/julienkay/MobileNeRF-Unity-Viewer/blob/2011cd84302ece6f1e0a31dd6f3479b8ad4c6f80/Editor/MobileNeRFImporter.cs#L28-L59

**Comment by yosun** [comment]
Ahhhh I can try it on windows

**Comment by polobymulberry** [comment]
@julienkay 
on mac, you can move menu item code "Import from disk" to top like this
![image](https://github.com/julienkay/MobileNeRF-Unity-Viewer/assets/5304430/b3df3bd1-25e8-4c20-bbae-5a2723ddd0a2)

**Comment by julienkay** [comment]
Someone proposed a fix for this. Let me know if you keep having problems on macOS.
